app/components/export/export.js

```diff
@@ -184,6 +189,26 @@ export default translate()(class Export extends Component {
             <a onClick={() => this.setDateRange(14)}>Last 14 Days</a>
           </div>
 
+          <div className="Export-units">
+            Units:
+            <input
+              name="bgUnits"
+              type="radio"
+              value={MGDL_UNITS}
+              checked={this.state.bgUnits === MGDL_UNITS}
+              onChange={this.handleInputChange}
+            />{' '}
+            {MGDL_UNITS}
+            <input
+              name="bgUnits"
+              type="radio"
+              value={MMOLL_UNITS}
+              checked={this.state.bgUnits === MMOLL_UNITS}
+              onChange={this.handleInputChange}
+              />{' '}
```

Comment:
Tiniest of nitpicks: Indentation on this closing bracket not aligned with the previous input

Reply:
So it is! Now that I see it, it bothers me too :) Fixing.